### PR TITLE
fixes wso2/product-iots#1246

### PR DESCRIPTION
**Pull request by lasanthaDLPDS**

### CODE REVIEW

**Review by geethkokila — APPROVED** [review]

File: components/device-mgt/org.wso2.carbon.device.mgt.ui/src/main/resources/jaggeryapps/devicemgt/api/device-api.jag [hunk 1]
```diff
@@ -73,6 +73,31 @@ if (!user) {
         } else {
             result = 400;
         }
+    }else if (uriMatcher.match("/{context}/api/devices/sketch/generate_link")) {
```
Comment (madhawap) [inline components/device-mgt/org.wso2.carbon.device.mgt.ui/src/main/resources/jaggeryapps/devicemgt/api/device-api.jag:76]: Need a space after the curly brace. Please format the code "} else if (...) {"

File: components/device-mgt/org.wso2.carbon.device.mgt.ui/src/main/resources/jaggeryapps/devicemgt/api/device-api.jag [hunk 2]
```diff
@@ -73,6 +73,31 @@ if (!user) {
...
+
+        if (!request.getContent()){
+            log.error("Request Payload Is Empty");
+            // HTTP status code 400 refers to - Bad request.
+            result = 400;
+        }else{
```
Comment (madhawap) [inline components/device-mgt/org.wso2.carbon.device.mgt.ui/src/main/resources/jaggeryapps/devicemgt/api/device-api.jag:82]: Here also need a space after the curly brace and the key word 'else'. Please format the code "} else {"

File: components/device-mgt/org.wso2.carbon.device.mgt.ui/src/main/resources/jaggeryapps/devicemgt/api/device-api.jag (hunk 1 above)
Comment (lasanthaDLPDS) [reply components/device-mgt/org.wso2.carbon.device.mgt.ui/src/main/resources/jaggeryapps/devicemgt/api/device-api.jag:76]: This is fixed 0ce103e1ad33617f640d5c03b57477f2b5adb4db

File: components/device-mgt/org.wso2.carbon.device.mgt.ui/src/main/resources/jaggeryapps/devicemgt/api/device-api.jag (hunk 2 above)
Comment (lasanthaDLPDS) [reply components/device-mgt/org.wso2.carbon.device.mgt.ui/src/main/resources/jaggeryapps/devicemgt/api/device-api.jag:82]: This is fixed 0ce103e1ad33617f640d5c03b57477f2b5adb4db

**Pull request merged**